### PR TITLE
Add Pizza Clause (11.4)

### PR DESCRIPTION
**Pull request by Thomas-Malcolm**
Co-Authored-By: Tom <52238845+Thomas-Malcolm@users.noreply.github.com>

### CODE REVIEW

**Review by nicklambourne — CHANGES_REQUESTED** [review]
communityIcon_mcqfvwckzgk41

**Review by JamesDearlove — CHANGES_REQUESTED** [review]
We're gonna need some justification here team 🍕 

Also, I'd argue that garlic bread should be mandated in the constitution instead of Hawaiian pizza.

**Comment by katrinafyi** [comment]
I thought garlic bread was deprecated in favour of the so-called cheesy garlic scrolls.

**Review by sannibunny — CHANGES_REQUESTED** [review]
As a vegetarian I am offended that the mandated pizza has meat. As a human, I am offended by pineapple on pizza. I will be organising a coup shortly.

**Comment by bradleysigma** [comment]
> Australian Canadian Hawaiian Pizza.

File: constitution.md [hunk 1]
```diff
@@ -162,15 +162,19 @@ the membership shall consider at a General Meeting whether that person's members
...
 
-11.2 The management committee may exercise all the power of the Society:  
+11.2 The management committee may exercise all the power of the Society:
 * to raise or secure the payment of money in such manner as the members of the Society may think fit and secure the payment or performance of any debt, liability or other engagement incurred or to be entered into by the Society in any way.
 
 11.3 The management committee will take full responsibility for all publications produced by the Society or by any of its members.
 
+11.4 At any event, function or gathering where UQCS is a financial contributor and pizza is offered, at least one (1) Hawaii\[a\]n pizza where:
+* a Hawaii\[a\]n pizza is a pizza originating in Canada, and is traditionally topped with pineapple, tomato sauce, cheese, and either ham or bacon [(source)](https://en.wikipedia.org/wiki/Hawaiian_pizza).
+* where procuring a Hawaii\[a\]n pizza from Canada is unfeasible, a substitute pizza originating from an establishment in Australia will suffice, provided that it meets the other requirements in 11.4.
```
Comment (49Indium) [inline constitution.md:176]: The phrase "a substitute pizza originating from an establishment in Australia will suffice, provided that it meets the other requirements in 11.4" is a bit concerning. There are no other requirements in 11.4, as the toppings listed are only "traditionally" supplied and the Canadian origin can be bypassed. Nothing is stopping UQCS executive from calling a meat-lovers pizza ("originating from an establishment in Australia") by the term "Hawaian".

**Review by 49Indium — CHANGES_REQUESTED** [review]

**Comment by JamesDearlove** [comment]
Tabled

**Comment by LimaoC** [comment]
Withdrawn at 2023 AGM.

**Pull request closed without merging**